bin/i18n/sync-codeorg-out

```diff
@@ -82,4 +83,105 @@ def rebuild_blockly_js_files
   end
 end
 
+# retrieves an ordered array of strings representing all image or link
+# values in the passed value; can support strings, arrays, or
+# (recursively) hashes. For hashes, results will be lexicographically
+# ordered by key.
+def get_links_and_images(value)
+  # regex to detect links and images in both markup and markdown formats
+  link_or_image_re = /(?:\]\(\S*\)|(?:src|href)=["']\S*["'])/
+
+  if value.nil?
+    return []
+  elsif value.instance_of? String
+    return value.scan(link_or_image_re)
+  elsif value.instance_of? Array
+    return value.join(' ').scan(link_or_image_re)
+  elsif value.instance_of? Hash
+    values = value.keys.sort.map do |key|
+      get_links_and_images(value[key])
+    end
+    return values.flatten
+  end
+end
+
+def check_for_bad_translations
+  categories_to_check = %w(
```

Comment:
Would it make sense to check all dashboard categories? May be good to catch them inserting links we don't expect since we don't sanitize everything yet. Or overkill?